app/models/grouping.rb

```diff
@@ -694,6 +694,23 @@ def access_repo
     end
   end
 
+  def get_next_grouping(going_right, current_role)
```

Comment:
Okay, just a couple of quick cleanup things now!

1. Add some documentation for this function.
2. `going_right` isn't a great name. The standard convention for this sort of behaviour is to name it `reversed` (so that `true` causes the method to go backward, and `false` goes forward).
3. Also, it would be good to make `going_right` the second parameter. In terms of function design, we try to order parameters in (rough) order of importance. Typically boolean flags are not very important (and in this case it's less important than the role needed to get the groupings).